carla_ackermann_control_wrapper/CMakeLists.txt

```diff
@@ -100,7 +103,7 @@ target_link_libraries(${PROJECT_NAME}
 #############
 
 ## Mark executables and/or libraries for installation
-install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_lib
+install(TARGETS ${PROJECT_NAME}_node
```

Comment:
Please add the ${PROJECT_NAME}_lib here as well. I think not having it could lead to a linker error at runtime.

Reply:
Added